jr-dev-training.md

```diff
@@ -0,0 +1,101 @@
+# Jr Developer Training Schedule
+
+Welcome to the team! We are very happy to work with you! We want you to feel comfortable coming to any of us with questions. Remember one of the dev team values is: **SHOUT for help**. Don't spend hours turmoiling over an issue. Bubble it up quickly and loudly. That said, it may be hard know what you don't know. This guide exists to help train you up quickly.
+
+You will go through each of these sections with your work buddy. Each day will consist of:
+
+* 1 hour of lecture
+* 3 - 4 hours of self study
+
+
+The rest of your day should be spent contributing to live code. After all, the best way to learn is to dive in and do! Your work buddy will get you into our Jira system, find some good tickets for you to work, and assign them to you.
+
+## Training Schedule
+
+### Day 1
+#### Routes, MVC, Rails, and Heroku
+
+1. Web requests, REST, and routes
+1. Error driven development
+1. Building Controllers from scratch
+1. Generators vs Scaffolds
+1. Building Views from scratch
+1. ERB is gross. Lets learn HAML!
+1. Deploying to Heroku
+1. Common Heroku commands/debugging
+
+##### Homework/Self Study
+Spin up a new rails app to blog about your first few days here! The app should have a Posts Table with `title:string` and `body:text` attributes. It should also have a Comments Table with references to the posts tale _(a post can have many comments)_.
+
+**Do NOT** use scaffolding/generators for your routes, views, or controllers. You can generate your models because we want the timestamped migration files built for us, but that is it!
```

Comment:
Like evar or just in this exercise?

Reply:
@moottoast - Just in this exercise. I'm great with them developing code however they want, but I need them to understand it all at first. Just what happens when you make a controller that doesn't inherit from `ApplicationController` or what the flow from route -> controller -> view is.

Reply:
Also agree that this could get stale after a while. Current plan is to update this with every few new hires (based on feedback/failures with the current hires we walk them through)

Reply:
👍

Reply:
👍